SYCL/Matrix/joint_matrix_tensorcore.cpp

```diff
@@ -163,6 +163,14 @@ void test() {
                               accB.get_pointer() + (k * K * Big_N) + (n * N),
                               Big_N);
 
+            // Convert values if using tf32
+            if constexpr (std::is_same<T3, precision::tf32>::value) {
+              for (auto i = 0; i < 4; ++i) {
```

Comment:
I think it would be OK to merge this test as it is: then I can merge it into https://github.com/intel/llvm-test-suite/pull/975 and use the element wise interface to address your comment: adding a comment on where 4 comes from would then not be necessary.

Reply:
This PR doesn't change the way that element wise ops could be performed in current sycl tip, it is just adding a new free tf32 rounding function and tf32 joint_matrix implementation. I don't think there is anything implicitly wrong with it being merged before #5964. I think It is easier to merge this one first because @hdelan is currently on holiday anyway and I know how to adjust #5964 to take account if these tf32 changes easily. Probably https://github.com/intel/llvm/pull/5964 will be ready to be merged then as well anyway by the sound of things.

Reply:
No but https://github.com/intel/llvm/pull/5870 is approved.

Reply:
I see, we agreed on addressing the spec changes in a separate PR (https://github.com/intel/llvm/pull/5870#pullrequestreview-950146811)
In this case, this test can also be approved and merged. Thank you for the clarifications.